src/qt/locale/bitcoin_nl.ts

```diff
@@ -1457,45 +1434,33 @@
         <source>Always do your own research before using an external cryptocurrency service</source>
         <translation>Doe altijd je eigen onderzoek voordat je een externe cryptocurrency-service gebruikt</translation>
     </message>
-    <message>
-        <source>Never share your private key to an untrustworthy person.</source>
-        <translation type="vanished">Deel uw privésleutel nooit met een onbetrouwbaar persoon.</translation>
-    </message>
-    <message>
-        <source>Who own the private keys own the coins.</source>
-        <translation type="vanished">Wie de privésleutels bezit, bezit de munten.</translation>
-    </message>
-    <message>
-        <source>To see ongoing development and contribute, checkout Dogecoin repository on GitHub!</source>
-        <translation type="vanished">Bekijk de Dogecoin-repository op GitHub om de lopende ontwikkeling te zien en bij te dragen!</translation>
-    </message>
     <message>
         <source>Services that claim to double your dogecoins are always ponzi schemes</source>
         <translation>Diensten die beweren uw dogecoins te verdubbelen zijn altijd ponzifraude</translation>
     </message>
     <message>
         <source>Never share your wallet.dat file with anyone</source>
-        <translation type="unfinished"></translation>
+        <translation>Deel uw wallet.dat bestand nooit met iemand</translation>
     </message>
     <message>
         <source>For advanced operations, use the console in &apos;Help&apos; -&gt; &apos;Debug Window&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Gebruik voor meer geavanceerde functies de console in &apos;Help&apos; -&gt; &apos;Foutopsporingsvenster&apos;</translation>
```

Comment:
Voor meer geavanceerde functies, gebruik de console in &apos;Help&apos; -&gt; &apos;Foutopsporingsvenster&apos;